bin/cli.js

```diff
@@ -157,7 +157,7 @@ function performRelease (options) {
 }
 
 function handleError (err) {
-  const msg = err.message || JSON.stringify(err)
+  const msg = err.message || JSON.stringify(err, null, '  ')
```

Comment:
```suggestion
  const msg = err.message || JSON.stringify(err, null, 2)
```
2 does same thing and is less invisible

Reply:
oh wild